features/sd-step.feature

```diff
@@ -0,0 +1,45 @@
+@sd-step
+Feature: Shared Steps
+
+    Users want to be able to share code for building software, so that my builds are
```

Comment:
Users want to be able to share code for building software, so that `their` builds are less prone to drift.